src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -47,7 +47,6 @@ class SQLiteAdapter extends PdoAdapter
         self::PHINX_TYPE_CHAR => 'char',
         self::PHINX_TYPE_DATE => 'date_text',
         self::PHINX_TYPE_DATETIME => 'datetime_text',
-        self::PHINX_TYPE_DECIMAL => 'decimal',
```

Comment:
Are you sure we need to remove this type here? It can have other side effects and regressions.

Reply:
Indeed that line of code should not be removed.

I reversed the change.